client/my-sites/sharing/connections/services/google-my-business.js

```diff
@@ -25,40 +32,77 @@ export class GoogleMyBusiness extends SharingService {
 	};
 
 	// override `createOrUpdateConnection` to ignore connection update, this is only useful for publicize services
-	createOrUpdateConnection = () => {};
+	createOrUpdateConnection = ( keyringConnectionId, externalUserId = 0 ) => {
+		this.props
+			.saveSiteSettings( this.props.siteId, {
+				google_my_business_keyring_id: keyringConnectionId,
```

Comment:
We can make it an separate action in a follow up PR (once we have a `state/google-my-business/actions.js` file)

Reply:
FYI, this `action.js` file is now available in `master`.